Gems/Atom/RHI/Vulkan/Code/Source/RHI/WSISurface.h

```diff
@@ -27,6 +27,7 @@ namespace AZ
             struct Descriptor
             {
                 RHI::WindowHandle m_windowHandle;
+                void* m_nativeConnection;
```

Comment:
Needs to be init to nullptr

Reply:
Use SDL instead of xcb create windows is a better choose, m_nativeConnection is not neeed.